README.md

```diff
@@ -1,27 +1,29 @@
 [![Build Status](https://travis-ci.org/stripe/veneur.svg?branch=master)](https://travis-ci.org/stripe/veneur)
 [![GoDoc](https://godoc.org/github.com/stripe/veneur?status.svg)](http://godoc.org/github.com/stripe/veneur)
 
-Veneur (venn-urr) is a server implementation of the [DogStatsD protocol](http://docs.datadoghq.com/guides/dogstatsd/#datagram-format), which is a superset of the StatsD protocol.
+Veneur (venn-urr) is a server implementation of the [DogStatsD protocol](http://docs.datadoghq.com/guides/dogstatsd/#datagram-format) for aggregating metrics and sending them to downstream storage, typically [Datadog](http://datadoghq.com). It can also as a [global aggregator](#global-aggregation) for histograms, sets and counters.
+
+More generically, Veneur is a convenient, host-local sink for various observability primitives.
 
 # Status
 
-Veneur is currently handling all metrics for Stripe and is considered production ready. It is, however, still in heavy development and may change!
+Veneur is currently handling all metrics for Stripe and is considered production ready. It is under active development and maintenance!
 
 Building veneur requires Go 1.7 or later.
 
 # Motivation
 
 We wanted percentiles, histograms and sets to be global. Veneur helps us do that!
 
-Veneur is a DogStatsD implementation that acts as a local collector and — when configured differently — as an aggregator for
+Veneur is a DogStatsD implementation that acts as a local collector and — optionally — as an aggregator for
 some metric types, such that the metrics are *global* rather than host-local. This is particularly useful for histograms,
 timers and sets, as in their normal, per-host configuration the percentiles for histograms can be less effective or even
 meaningless. Per-host unique sets are also often not what's desired.
 
 Global \*StatsD installations can be problematic, as they either require client-side or proxy sharding behavior to prevent an
 instance being a Single Point of Failure (SPoF) for all metrics. Veneur aims to solve this problem. Non-global
-metrics like counters and gauges are collected by a local Veneur instance and sent to Datadog at flush time. Global metrics (histograms and sets)
-are forwarded to a central Veneur instance for aggregation before being sent to Datadog.
+metrics like counters and gauges are collected by a local Veneur instance and sent to storage at flush time. Global metrics (histograms and sets)
```

Comment:
Can we say `sent to storage (e.g. Datadog) at flush time`?


As it stands, the first mention of Datadog in the new README is halfway down the page, and in a context where it doesn't make sense. If we add this, though, we don't need to change anything else in the rest.